.github/actions/e2e/create-cluster/action.yaml

```diff
@@ -49,7 +49,7 @@ runs:
     run: AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_LOCATION=${{ inputs.location }} make az-create-workload-msi
   - name: create cluster
     shell: bash
-    run: AZURE_CLUSTER_NAME=${{ inputs.cluster_name }} AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_ACR_NAME=${{ inputs.acr_name }} AZURE_LOCATION=${{ inputs.location }} make az-mkaks-cilium
+    run: AZURE_CLUSTER_NAME=${{ inputs.cluster_name }} AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_ACR_NAME=${{ inputs.acr_name }} AZURE_LOCATION=${{ inputs.location }} make az-mkaks-cniv1
```

Comment:
The simple solution here is to just add create-cluster steps for each networking configuration makefile command. With the new create cluster commands I add here we can run e2es for 
Static Azure CNI 
Azure CNI with Overlay 
Azure CNI with Overlay with Cilium 


The real question is should we not? Should we instead add parameterized create cluster steps in the e2e tests themselves?